doc/2-interface/export.md

```diff
@@ -1,26 +1,27 @@
 # export
 
-Furnace allows you to export your song in several formats. this section deals with describing the available export options.
+Furnace allows you to export your song in several formats. this section deals with describing the available export options within each tab of the export window.
 
-## export audio
+## Audio
 
 this option allows you to export your song in .wav format. I know I know, no .mp3 or .ogg export yet, but you can use a converter.
 
-there are two parameters:
-
+- **Export type**:
+  - **one file**: exports your song to one .wav file.
+  - **multiple files (one per chip)**: exports the output of each chip to .wav files.
+  - **multiple files (one per channel)**: exports the output of each channel to .wav files.
+    - useful for usage with a channel visualizer such as corrscope.
+- **Bit depth**:
+  - **16-bit integer**: the most common type of .wav file, perfect for music playback.
+  - **32-bit float**: used for advanced audio manipulation. don't select this unless you know what you need it for.
+- **Sample rate**: the default is 44100, the most common .wav sample rate.
```

Comment:
Does not explain how does sample rate affect quality.

Reply:
I hope the new content works.